### PR TITLE
Simple plan to test the table and diode

### PR DESCRIPTION
**Pull request by stan-dot**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         runs-on: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
```
Comment (DiamondJoseph) [inline .github/workflows/ci.yml]: Make these another PR, it's not related

Comment (stan-dot) [reply .github/workflows/ci.yml]: ok

File: src/i18_bluesky/plans/simple_plans.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        "steps" + prefix: int(scan_args[2]),
+    }
+
+
+def step_scan(
+    detectors: List[StandardDetector],
+    motor: Motor,
+    scan_args: List[object],
+    metadata: Optional[Mapping[str, Any]] = None,
+) -> MsgGenerator:
```
Comment (DiamondJoseph) [inline src/i18_bluesky/plans/simple_plans.py:27]:
```suggestion
def step_scan(
    detectors: List[StandardDetector],
    motor: Motor,
    *scan_args: tuple[Motor, float, float, int],
    *,
    metadata: Optional[Mapping[str, Any]] = None,
) -> MsgGenerator:
```

As I understand it, this makes scan_args take an arbitrary number of tuples of that form, has metadata as kw-only argument.

File: src/i18_bluesky/plans/simple_plans.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Dictionary of scan arguments from motor scan_args (start, stop, steps)
+"""
+
+
+def make_args(motor, scan_args, prefix=""):
+    return {
+        "motor" + prefix: motor,
+        "start" + prefix: scan_args[0],
+        "stop" + prefix: scan_args[1],
+        "steps" + prefix: int(scan_args[2]),
```
Comment (DiamondJoseph) [inline src/i18_bluesky/plans/simple_plans.py:18]: There's generally 2 ways of defining a plan, either with a fixed step length between points or a number of points. Here you're mixing the two concepts. If you're going to do number of points, stick with `num` rather than `steps` to make it clearer what's going on.

e.g. 

start=0, stop=5, step=2 => {0, 2, 4}
start=0, stop=5, num=2 => {0, 2.5, 5}

(step purposefully chosen to show why I prefer `num`, but users like to think in terms of a fixed step size)

**Review by DiamondJoseph — COMMENTED** [review]
I can't request changes because you merged already :upside_down_face:

File: src/i18_bluesky/plans/simple_plans.py (hunk 3 above)
Comment (stan-dot) [reply src/i18_bluesky/plans/simple_plans.py:18]: yeah I thought I need a review but then I saw this does not need a review so I could not resist the big green button.

this is all just drafts anyway, won't be run on the beamline until the 9th

**Comment by stan-dot** [comment]
thank you for your comments @DiamondJoseph
